### ISSUE TITLE
Cannot delete tag key from GUI

### ISSUE THREAD

**Issue by dkuji** [distribworks/dkron#1331]
I use the GUI to edit a job.
When I delete a tag with the delete button on the edit screen, the value is removed but the key remains.

<img width="336" alt="ScreenShot 2023-05-18 11 12 34" src="https://github.com/distribworks/dkron/assets/22391144/cefad142-00eb-4ad2-9cfa-b7e4be5993e6">

Is there any way to delete the key from the GUI?

**Comment by vcastellm** [comment]
Good catch, thanks for reporting, it looks like a bug in UI, you should use the API at this time until we release a fix.

**Comment by cobolbaby** [comment]
The issue has been fixed in version 4.0.0-beta2.
